vcpkg.json.in

```diff
@@ -177,6 +177,14 @@
         {
             "name": "zlib",
             "platform": "@VCPKG_ZLIB@"
+        },
+        {
+            "name": "opentelemetry-cpp",
+            "features": [
+                "otlp",
+                "otlp-http",
+                "otlp-grpc"
```

Comment:
Are you sure you need the "grpc" support (its typically an alternative to http)?

Reply:
is it problematic?
I've been asked to support OTLP, it's not clear if it'll use http or not.